packages/blade/src/components/Amount/__tests__/Amount.web.test.tsx

```diff
@@ -80,10 +82,14 @@ describe('<Amount />', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('should render MYR currency Amount ', () => {
-    const { container } = renderWithTheme(<Amount currency="MYR" value={1000} />);
-    expect(container).toMatchSnapshot();
-  });
+  for (const currency of Object.keys(currencyPrefixMapping)) {
```

Comment:
That's a lot of snapshots! Since currency is just string, lets just check for 1 or 2 currencies?

Reply:
Makes sense, updated.